object.go

```diff
@@ -630,6 +634,30 @@ func (obs *obs) Delete(name string) error {
 	return obs.js.purgeStream(obs.stream, &StreamPurgeRequest{Subject: chunkSubj})
 }
 
+func publishMeta(info *ObjectInfo, js JetStreamContext) error {
+	// Prepare the meta message
+	metaSubj := fmt.Sprintf(objMetaPreTmpl, info.Bucket, encodeName(info.ObjectMeta.Name))
+	mm := NewMsg(metaSubj)
+	mm.Header.Set(MsgRollup, MsgRollupSubject)
+
+	info.ModTime = time.Time{} // We don't store this, just do it after publish
+	data, err := json.Marshal(info)
+	if err != nil {
+		return err
+	}
+
+	// Publish the meta message.
+	mm.Data = data
+	_, err = js.PublishMsg(mm)
```

Comment:
For short calls like this, when we really only care about the error, I would suggest to inline the call to `PublishMsg()`:
```go
if _, err := js.PublishMsg(mm); err != nil {
    return err
}
```

Reply:
Fixing

Reply:
fixed